pyaedt/q3d.py

```diff
@@ -709,6 +709,425 @@ def export_matrix_data(
                 self.logger.error("Export of matrix data was unsuccessful.")
                 return False
 
+    def export_equivalent_circuit(
+        self,
+        file_name,
+        setup_name=None,
+        sweep=None,
+        variations=None,
+        matrix_name=None,
+        num_cells=2,
+        user_changed_settings=True,
+        include_cap=True,
+        include_cond=True,
+        include_dcr=True,
+        include_dcl=True,
+        include_acr=True,
+        include_acl=True,
+        include_r=True,
+        include_l=True,
+        add_resistance=True,
+        include_cpp=False,
+        parse_pin_names=False,
+        cs="Global",
+        export_distributed=True,
+        lumped_length="1meter",
+        rise_time_value=None,
+        rise_time_unit=None,
+        coupling_limit_type=None,
+        cap_limit=None,
+        ind_limit=None,
+        res_limit=None,
+        cond_limit=None,
+        model_name=None,
+        freq=0,
+        file_type="HSPICE",
+    ):
+        """Export matrix data.
+
+        Parameters
+        ----------
+        file_name : str
+            Full path to save the equivalent circuit to.
+            Options for file extensions are: .cir, .sml, .sp, .pkg, .spc, .lib, .ckt, .bsp, .dml, .icm.
```

Comment:
```suggestion
            Options for file extensions are CIR, SML, SP, PKG, SPC, LIB, CKT, BSP, DML, and ICM.
```

Reply:
Google Developer Style Guide, which we are following, and also the Ansys Style Guide, says to use all capital letters in reference to file extensions when not using an entire file name (such as ``MatrixData1.cir``, which is formatted as a literal string.

Reply:
good to know, thanks!